src/microbe_stage/MicrobeAI.cs

```diff
@@ -367,6 +367,14 @@ private void EngagePrey(Vector3 target, Random random, bool engulf)
     // For doing run and tumble
     private void RunAndTumble(Random random)
     {
+        // If this microbe is actually immobile, just initialize by moving in a random direction.
```

Comment:
```suggestion
        // If this microbe is currently stationary, just initialize by moving in a random direction.
```

Being picky with English, but normally "immobile" means that something CAN'T move, as opposed to not currently moving.

Reply:
If I may say, you really should consider leaving us a memo when you change the meaning of the words you borrow from us. :p Actually was actually intended, but both case works so I'll just leave it like this. Thanks!

Reply:
I'll get back to you with a list of all culture America has ruined, but that might be beyond the scope of this PR.